File: LogicBulding/21JulyFor/prog1.c


#include<stdio.h>

void main() {

	for(int i = 0; i < 7; i++) {

		printf("%c %c ",'A'+i,'z'-i);

	}

}	
/*
vivek@vivek-HIRAY:~/Core2Web-TREE-/LogicBulding/21JulyFor$ cc prog1.c 
vivek@vivek-HIRAY:~/Core2Web-TREE-/LogicBulding/21JulyFor$ ./a.out 
A z B y C x D w E v F u G t 
*/

File: CodeSnippets/javaSnippets/5Aug/prog1.java


class Core2web {

	static int var = 2;
	public static void main(String[] args) {

		var = (var=4)*(++var);
		System.out.println(var);

	}

}
/*vivek@vivek-HIRAY:~/Core2Web-TREE-/CodeSnippets/javaSnippets/5Aug$ javac prog1.java 
vivek@vivek-HIRAY:~/Core2Web-TREE-/CodeSnippets/javaSnippets/5Aug$ java Core2web 
20
*/
